src/utils/filterResults.ts

```diff
@@ -73,7 +73,7 @@ export default function filterResults (unsortedPropertyResults: UnprocessedResul
             const canAddAsNeighbour = matchTallies.neighbour < maxNumNeighbours
             const canAddBasedOnFilters = propertyGroupMatch || propertyTypeMatch
             const isManaged = (property.Managed === managed) || managed === 'None'
-            const shouldAddProperty = isManaged || canAddBasedOnFilters || canAddAsNeighbour
+            const shouldAddProperty = isManaged && (canAddBasedOnFilters || canAddAsNeighbour)
```

Comment:
This was causing too many properties to be added. This logic seems more in line with what they originally wanted.